File: desktop/src/com/netrew/desktop/DesktopLauncher.java
package com.netrew.desktop;

import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application;
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration;
import com.netrew.Main;

public class DesktopLauncher {
	public static void main (String[] arg) {
		Lwjgl3ApplicationConfiguration config = new Lwjgl3ApplicationConfiguration();
		config.setTitle("Netrew");
		config.setWindowedMode(1600, 1200);
		new Lwjgl3Application(new Main(), config);
	}
}

File: desktop/src/com/netrew/desktop/DesktopLauncher.kt
package com.netrew.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.netrew.Main

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.title = "Netrew"
        config.width = 1600
        config.height = 1200
        LwjglApplication(Main(), config)
    }
}
